packages/cli/generators/project/templates/README.md.ejs

```diff
@@ -1,3 +1,52 @@
 # <%= project.name %>
 
-[![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+<% if (project.description) { -%>
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [Loopback CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
+
+## First steps
+
+### Starting the application
+
+Starting the application is as easy as:
+
+```sh
+npm start
+```
+
+### Using CLI Commands
+
+Refer to [Command-line interface](https://loopback.io/doc/en/lb4/Command-line-interface.html) for the full list of LoopBack 4 commands.
+
+### Adding Additional Dependencies
```

Comment:
Do we want to mention `lb4 upgrade` to handle project dependency upgrade for newer LB versions?

Reply:
It is currently documented at this [link](https://loopback.io/doc/en/lb4/Command-line-interface.html)